src/pages/team/index.tsx

```diff
@@ -241,11 +240,7 @@ const IconLink: FunctionComponent<{className?: string}> = ({
 export const getServerSideProps: GetServerSideProps<TeamPageProps> = async function (
   context: any,
 ) {
-  const {eggheadToken} = getTokenFromCookieHeaders(
-    context.req.headers.cookie as string,
-  )
```

Comment:
The `getAccessTokenFromCookie` call works for client-side requests. In these server-side requests where we call `getTokenFromCookieHeaders`, you need to make the call where you have access to the `context.req`.

Perhaps the thing to do is for `getGraphQLClient` to accept an optional `token` argument and fallback to `getAccessTokenFromCookie` if that isn't passed in.

Reply:
yes, that is likely the case

if a token is passed in, use it. If it isn't, check the client cookies and use that if available